### DISCUSSION TITLE
EteSync support

### DISCUSSION THREAD

**Discussion by StoppingBuck** [alainm23/planify#1632] [Ideas]
EteSync (https://www.etesync.com/) is a new'ish protocol for synchronizing calendar, contacts and tasks. On Android, apps like https://tasks.org support the protocol, but there are currently no desktop apps supporting it. It could be cool if Planner could be the first :)

**Comment by phillipprado** [comment]
I too would love this feature and have been hoping it would be implemented for some time now. EteSync offers E2E encryption, it's open source, and it's more affordable than Todoist. I feel like this integration would go over well with the community, and there still aren't any Linux desktop task apps that I know of that offer this functionality.

**Comment by smac89** [comment]
You can achieve etesync support on your own by installing the [etesync-dav](https://github.com/etesync/etesync-dav) adapter and login, then use the url  and password you are given to log into planner.

**Comment by phillipprado** [comment]
The caldav option isn't available in Planify from what I can tell, and the EteSync caldav bridge won't work properly with Planner for me. I've tried a few different times on a few different machines. I can get it to work on other apps, just not Planner.

**Comment by smac89** [comment]
It worked for me, as I can log in, but I found the experience to be very slow and ended up abandoning the idea

**Comment by CristianKerr** [comment]
Hi there, I just tried the new dav implementation in Planify, but it doesn't work with eteSync (expected). 
I would love if it was supported and if this is implemented and I will be able to use the app, you will have a new patreon/donator for sure (me)! ❤

**Comment by shvchk** [comment]
Tried to use etesync-dav with Planify. Initial sync goes fine, but doesn't work after that: tasks created/updated in Planify do not appear on the server and vice versa. Tried setting server URL both as http://localhost:37358/ and http://localhost:37358/username/

Using Chiri for now, which works well with etesync-dav (see [this FR for native EteSync support for details](https://github.com/SapphoSys/chiri/issues/40)).

I also have high hopes for the [upcoming Errands rewrite in C](https://github.com/mrvladus/Errands/pull/427).

**Comment by alainm23** [comment]
EteSync works with Planify via the [etesync-dav](https://github.com/etesync/etesync-dav) proxy — task sync works great in both directions. The only limitation is that creating and deleting task lists must be done from the EteSync app, not from Planify.EteSync works with Planify via the [etesync-dav](https://github.com/etesync/etesync-dav) proxy — task sync works great in both directions. The only limitation is that creating and deleting task lists must be done from the EteSync app, not from Planify.
